io/src/main/java/org/commonjava/maven/ext/manip/io/SettingsIO.java

```diff
@@ -47,7 +48,7 @@ public void write( Settings settings, File settingsFile )
     {
         try
         {
-            PrintWriter printWriter = new PrintWriter( settingsFile );
+            PrintWriter printWriter = new PrintWriter( settingsFile , "UTF-8" );
```

Comment:
Yea, it was missed after trying the private constructor <code>private PrintWriter(Charset charset, File file)</code>, It's done. Thanks!